distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numeric/Math_Context.enso

```diff
@@ -0,0 +1,26 @@
+import project.Data.Numeric.Rounding_Mode.Rounding_Mode
+from project.Data.Numbers import Integer
+
+polyglot java import java.math.MathContext
+
+## A wrapper around the Java
+   [`MathContext`](https://docs.oracle.com/javase/8/docs/api/java/math/MathContext.html)
+   class.
+
+   `Math_Context` is used to specify finite or infinite precision requirements
+   to operations that construct `Decimal` values, or that compute them from
+   other values. It can also specify the `Rounding_Mode` to use when rounding is
+   used to meet precision requirements. See `Decimal` for more details.
+type Math_Context
+    ## PRIVATE
+    Value (math_context:MathContext)
+
+    ## Construct a `Math_Context` value.
+
+       Arguments:
+       - precision: The non-negative int precision setting. A value of `0`
+         specifies exact operations.
```

Comment:
IMO such 'magic' numbers should be discouraged in Enso.

I'd consider introducing a `type Exact` and making this `precision : Integer | Exact`. Then in user's code `precision=Exact` is much clearer what it means than `precision=0`.

Reply:
Done; used `Unlimited` to match java `BigDecimal`